### PR TITLE
feat: move translation resources to external lib

### LINKED ISSUE #1775: Move translation resources to separate library
Translations are used in the storefrontlib and in the core. But currently they are stored in storefrontlib. We should consider moving them into separate library.

It can be `@spartacus/assets` (then we can add there icons, fonts, etc. in future)

### PR DESCRIPTION
**Pull request by mirkosoft**
* i18n resources has been moved to new assets lib
* lang suffix has been removed from the chunks filenames
* created barrel for english translations
* prepared release/CI configuration to @spartacus/assets

PR with updates to installation script: https://stash.hybris.com/projects/C3PO/repos/spartacus-installation/pull-requests/24/overview

closes #1775 #1847 #2569

### CODE REVIEW

File: projects/assets/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Assets
```
Comment (marlass) [inline projects/assets/README.md]: We should override default README with a description of this particular library.

File: projects/assets/karma.conf.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      require('karma-coverage-istanbul-reporter'),
+      require('@angular-devkit/build-angular/plugins/karma'),
+    ],
+    client: {
+      clearContext: false, // leave Jasmine Spec Runner output visible in browser
+    },
+    coverageIstanbulReporter: {
+      dir: require('path').join(__dirname, '../../coverage/assets'),
+      reports: ['html', 'lcovonly'],
+      fixWebpackSourcePaths: true,
```
Comment (marlass) [inline projects/assets/karma.conf.js]: Would be good to add minimal threshold

File: projects/assets/src/translations/index.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { en } from './en/index';
+
+interface TranslationResources {
```
Comment (Pucek9) [inline projects/assets/src/translations/index.ts]: Please consider move it to another file.

Comment (marlass) [reply projects/assets/src/translations/index.ts]: The lib is so simple that I think that is not required at the moment

**Pull request merged**